planning/development-strategy.md

```diff
@@ -25,86 +25,161 @@ A simple little website about trees.
 
 <!-- issue title -->
 
-## Site Title
+## Site Title (milestone)
 
 <!-- user story -->
 
-> "as a user I can read the title of the page"
+> "As a user I can read the title of the page"
 
 <!-- detailed description -->
 
-The site needs a clear title that's easy to read at the top of the page.
+<!--The site needs a clear title that's easy to read at the top of the page.-->
 
 <!-- acceptance criteria -->
 
 - [ ] _there is a prominent title at the top of the web page_
 
 <!-- code you think you will need -->
 
-### Site Title: HTML
+### Site Title HTML
 
-- Header with the web site's title
+- Header with the web site's title (issue)
 
-### Site Title: CSS
+### Site Title CSS
 
-- styling to center the title
+- styling to center the title(issue)
 
 ---
 
-## Introduction
+## Menu (milestone)
 
-> "As a user I can read an introduction to this project"
+> As a user I can read the details of the website from the Menu.
```

Comment:
We don't have this user story in our Backlog. We have only "As a plantsman I can easily understand what is this page about"

Reply:
Yeah...that I will update it